src/main/java/org/agbrothers/englishbot/processing/ProcessingExchange.java

```diff
@@ -47,11 +53,21 @@ public void setResponseMessageText(String responseMessageText) {
         this.responseMessageText = responseMessageText;
     }
 
-    public Map<String, String> getResponseButtons() {
+    public List<Map<String, String>> getResponseButtons() {
         return responseButtons;
     }
 
-    public void setResponseButtons(Map<String, String> responseButtons) {
+    public void setResponseButtons(List <Map<String, String>> responseButtons) {
         this.responseButtons = responseButtons;
+        addMainMenuButton(responseButtons);
+    }
+
+    public List<Map<String,String>> addMainMenuButton(List<Map<String, String>> result) {
```

Comment:
ProcessingExchange should be a POJO, since we use it for transferring data from request to response. It is not the right place to put a logic. This method should be in another place. Tip: consider `ProcessingCore`